### DISCUSSION TITLE
how to Register Renovate-approve in GitHub App

### DISCUSSION THREAD

**Discussion by AramSol** [renovatebot/renovate-approve-bot#171] [Q&A]
Hello everyone,

I'm currently exploring the auto-merge feature in Renovate. To enable this functionality, we require the approve bot. However, I'm unable to directly install any apps in our repository. As a workaround, I plan to install it in my personal account first and then propose it at the organization level. I'm in the process of registering Renovate-approve on my GitHub account, but I'm unsure about the permissions that should be granted to the registered app. Is there any example or guidance available on how to configure Renovate-approve during registration?

![image](https://github.com/renovatebot/renovate-approve-bot/assets/39582300/fd90512e-722a-435d-8d81-c2aa8bb93791)
![image](https://github.com/renovatebot/renovate-approve-bot/assets/39582300/6974741d-6358-4131-9231-f3d7db3f8be4)

BR
Aram

**Comment by rarkins** [comment]
The only hard part is permissions. To see which permissions are needed, you can take the first step to install the app we host and you see:

![image](https://github.com/renovatebot/renovate-approve-bot/assets/6311784/7f7653db-f0be-4dfc-84f8-a17ef020c971)

**Reply by AramSol** [reply]
this 2 permission are enough?
![image](https://github.com/renovatebot/renovate-approve-bot/assets/39582300/3348eaef-505f-4d78-82fa-85539c398204)

**Reply by rarkins** [reply]
You won't know unless you try, please don't ask us to confirm every step
